lib/providers/zmq.js

```diff
@@ -108,7 +112,7 @@ ZeroMqProvider.prototype._sendMessage = function(message) {
   var self = this;
 
   try {
-    self._pubSock.send([self._q, message]);
+    self._pubSock.send([self._q, self._q + ' ' + message]);
```

Comment:
Should add a comment here to explain why it is necessary to prepend the queue name/topic to the message.